libs/ReinforcerKit/tests/ReinforcerKit_test.cpp

```diff
@@ -70,35 +83,45 @@ TEST_F(ReinforcerkitTest, playSpinBlink)
 	auto expected_speed = 1;
 
 	EXPECT_CALL(mock_videokit, playVideoOnce);
-
-	{
-		InSequence seq;
-
-		EXPECT_CALL(mock_motor_left, spin(Rotation::counterClockwise, expected_speed));
-		EXPECT_CALL(mock_motor_right, spin(Rotation::counterClockwise, expected_speed));
-
-		EXPECT_CALL(mock_motor_left, stop());
-		EXPECT_CALL(mock_motor_right, stop());
-	}
+	EXPECT_CALL(mock_motor_left, stop).Times(1);
+	EXPECT_CALL(mock_motor_right, stop).Times(1);
+	EXPECT_CALL(mock_motor_left, spin(Rotation::counterClockwise, 1)).Times(1);
+	EXPECT_CALL(mock_motor_right, spin(Rotation::counterClockwise, 1)).Times(1);
+	EXPECT_CALL(mock_ears, setColor).Times(AnyNumber());
+	EXPECT_CALL(mock_ears, show).Times(AnyNumber());
+	EXPECT_CALL(mock_belt, setColor).Times(AnyNumber());
+	EXPECT_CALL(mock_belt, show).Times(AnyNumber());
```

Comment:
Okay, I didn't understand how LEDs are related to MotionKit, but it is not. You just fix the UNEXPECTED_CALL there.

Reply:
@HPezz can you put those in a function like we do in other tests:

```
	EXPECT_CALL(mock_ears, setColor).Times(AnyNumber());
	EXPECT_CALL(mock_ears, show).Times(AnyNumber());
	EXPECT_CALL(mock_belt, setColor).Times(AnyNumber());
	EXPECT_CALL(mock_belt, show).Times(AnyNumber());
```